mopidy/core/library.py

```diff
@@ -72,6 +72,22 @@ def browse(self, uri):
             return []
         return backend.library.browse(uri).get()
 
+    def get_images(self, uris):
+        """Lookup the images for the given URIs
+
+        :param list uris: list of uris to find images for
+        :rtype: dict mapping uris to :class:`mopidy.models.Image` instances
```

Comment:
I implemented it wrong, it should be `{uri: [images]}` - and yes to description can also be more clear.